infrastructure/application/index.ts

```diff
@@ -472,7 +464,7 @@ export = async () => {
           { name: "PORT", value: String(SHAREDB_PORT) },
           {
             name: "JWT_SECRET",
-            value: config.require("jwt-secret"),
+            value: config.requireSecret("jwt-secret"),
```

Comment:
Using `config`.requireSecret` make it so that pulumi forces you to encrypt this config value using `pulumi config set config-name --secret`. It's just nice so we don't end up storing secrets as plain-text.